dss-api.yml

```diff
@@ -2683,6 +2871,54 @@ definitions:
       - type
       - uuid
       - version
+  EventsManifest:
+    type: object
+    properties:
+      journal_id:
```

Comment:
This is returned but I don't see any way for the user to use the `journal_id` currently.  Are there future plans for users to be able to get a journal by id?

Reply:
There are no plans provide a get journal endpoint. However, `journal_id` is still useful if users wish to store journals locally.